phpBB/config/installer/container/services_converter_console.yml

```diff
@@ -0,0 +1,12 @@
+services:
+    console.converter.command.convert:
+        class: phpbb\install\console\command\converter\convert
+        arguments:
+            - '@installer.helper.config'
+            - '@installer.helper.container_factory'
+            - '@language'
+            - '@installer.helper.iohandler_factory'
+            - '@converter.module.obtain_data'
+            - '@phpbb.installer.converter.controller.helper'
+        tags:
+            - { name: console.installer.command}
```

Comment:
I think there is some issue with my YAMLs, last new lines arent getting saved. I had added a config for that in phpStorm. I'll have a look at theese. Thanks.